Equipment/EquipmentDb/Weapons/Ranged.xml

```diff
@@ -24,6 +24,19 @@
 		</source>
 	</melee_weapon>
 
+	<melee_weapon id="16966" phase="2">
+		<info name="Gorewood Bow" type="BOW" slot="RANGED" unique="no" req_lvl="56" item_lvl="62" quality="RARE" boe="no" icon="Inv_weapon_bow_04.png"/>
+		<dmg_range min="55" max="104" speed="2.5"/>
```

Comment:
Please add `<mutex item_id="16997"/>` and `<mutex item_id="16998"/>` so it is not possible to equip impossible setups.